app/frontend/courses/curricula/components/CoursesCurriculum__SubmissionBuilder.res

```diff
@@ -175,15 +181,36 @@ let make = (~target, ~targetDetails, ~addSubmissionCB, ~preview, ~checklist) =>
             />
           )
           |> React.array}
-      <div className={buttonClasses(state.checklist)}>
-        <Tooltip tip={tooltipText(preview)} position=#Start disabled={!isButtonDisabled(state)}>
-          <button
-            onClick={submit(state, send, target, targetDetails, addSubmissionCB)}
-            disabled={isButtonDisabled(state) || preview}
-            className="btn btn-primary flex justify-center grow md:grow-0">
-            {buttonContents(state.formState, checklist)}
-          </button>
-        </Tooltip>
+      <div>
+        {targetDetails->TargetDetails.allowAnonymous
+          ? <div className="mt-4 flex">
+              <input
+                onChange={_event => send(ToggleAnonymous)}
+                checked=state.anonymous
+                className="hidden checkbox-input"
+                id="anonymous"
+                type_="checkbox"
+              />
+              <label className="checkbox-label flex items-center" htmlFor="anonymous">
+                <span className="flex-shrink-0">
+                  <svg width="12px" height="10px" viewBox="0 0 12 10">
+                    <polyline points="1.5 6 4.5 9 10.5 1" />
+                  </svg>
+                </span>
+                <span className="text-sm"> {tr("submit_anonymous_label")->str} </span>
```

Comment:
I think it'd be helpful to add a description for this option. We could have a help-block below this label which reads something like: `You have the option to hide your name when participating in this discussion. If you're selecting this option, ensure you do not mention anything personally identifiable in your submission.`

Reply:
@harigopal 
https://github.com/pupilfirst/pupilfirst/pull/1531/commits/85dca763addd4c4065f535dad9a9701cf1fd42f8 I've updated the anonymous check-box help text.